package.json

```diff
@@ -248,6 +248,10 @@
                 "command": "hubspot.account.viewPersonalAccessKey",
                 "title": "Show personal access key info",
                 "icon": "$(link-external)"
+            },
+            {
+                "command": "hubspot.remoteFs.refresh",
+                "title": "Refresh"
```

Comment:
The extension manifest (`package.json`) contains the extension "contributions". Here we define the new refresh command, add an option for it in the "view/title" submenu for the remote fs (this is what adds it to the `...` button), add the `hubspot.treedata.remoteFs` view to our extension's sidebar, and add welcome content to display when the view is empty